actors/builtin/miner/miner_state.go

```diff
@@ -1110,7 +1111,16 @@ func (st *State) AdvanceDeadline(store adt.Store, currEpoch abi.ChainEpoch) (*Ad
 	}, nil
 }
 
-func MinerEligibleForElection(store adt.Store, mSt *State, thisEpochReward abi.TokenAmount, currEpoch abi.ChainEpoch) (bool, error) {
```

Comment:
Nit: this doesn't really belong with the miner actor any more, though I won't object much if you land it here.

I think it belongs directly under the `builtin` package, either in one of the existing files like `shared.go`, `network.go`, or a new one for node state utility functions.

Reply:
Filed in #1061 since this is something we can do post mainnet launch no problem.  I would have just cut and pasted but I remember from trying to write tests without the miner actor unit test harness that there is a bit of extra state juggling that isn't worth it right now.